varnish/tests/test_varnish.py

```diff
@@ -4,16 +4,39 @@
 
 import pytest
 
+from datadog_checks.dev.utils import get_metadata_metrics
+
 from . import common
 
 pytestmark = [pytest.mark.usefixtures('dd_environment'), pytest.mark.integration]
 
 
+GAUGE_IN_5_RATE_IN_6 = [
```

Comment:
does this mean that metrics in v5 and v6 are submitted as different types under the same name?

Reply:
exactly, the PR description explains it.

Reply:
Not anymore, I've split the pr, now is on https://github.com/DataDog/integrations-core/pull/8060